erratatool.yml

```diff
@@ -172,6 +172,8 @@ boilerplates:
     topic: |
       Red Hat OpenShift Container Platform release 4.14.z is now available with updates to packages and images that fix several bugs and add enhancements.
 
+      This release includes a security update for Red Hat OpenShift Container Platform 4.[y]
```

Comment:
We are hard coding the `[y]` to `14` elsewhere. Any reason to do that differently here?

Reply:
I think you are right. We can't CP this to 4.15 anyway so makes sense to hardcode it. I will update shortly.